framework/addons/utils.py

```diff
@@ -8,48 +8,42 @@
     'none': 'danger',
 }
 
+CAPABILITY_SET = [
+    'Permissions',
+    'View / download file versions',
+    'Add / update files',
+    'Delete files',
+    'Logs',
+    'Forking',
+    'Registering'
+]
+
 def read_capabilities(filename):
 
-    lines = open(filename).readlines()
-    lines = [
-        line.rstrip().split('\t')
-        for line in lines
-    ]
+    data_file = open(filename, 'r')
+    data = load_json(data_file.read())
 
-    addons = [
-        (col, cell.strip())
-        for col, cell in enumerate(lines[1])
-        if cell
-    ]
-    caps = [
-        (row, line[0].strip())
-        for row, line in enumerate(lines)
-        if line[0].strip()
-    ]
+    addons = data['addons']
+    disclaimers = data['disclaimers']
 
     rv = {}
```

Comment:
For consistency with the rest of the codebase, can you please rename this to `ret`?